src/index.js

```diff
@@ -68,11 +72,18 @@ async function run() {
     reviewers.push(...default_reviewers);
   }
 
+  core.info(`Possible Reviewers ${reviewers.join(', ')}, prepare filtering out already requested reviewers`);
+  reviewers = reviewers.filter((reviewer) => !current_reviewers.includes(reviewer));
+
   core.info('Randomly picking reviewers if the number of reviewers is set');
   reviewers = randomly_pick_reviewers({ reviewers, config });
 
-  core.info(`Requesting review to ${reviewers.join(', ')}`);
-  await github.assign_reviewers(reviewers);
+  if (reviewers.length) {
```

Comment:
```suggestion
  if (reviewers.length > 0) {
```

I think this should do an explicit inequality check